eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml

```diff
@@ -40,6 +40,7 @@ parameters:
 jobs:
 - job: ${{ parameters.GenerateJobName }}
   variables:
+    skipComponentGovernanceDetection: true
```

Comment:
I've been thinking about having a globals.yml file in common that could have some common properties. This might be one of them but I'm not sure I would do it just for this one case.

Reply:
It seemed to start popping up for java and adding 3+ minutes to their builds, hence the quick fix. But I agree next time we have to add something similar we can think about adding a globals.yml for eng/common.

Reply:
We already have a globals.yml. https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/variables/globals.yml

Do we want to serve another purpose

Reply:
@sima-zhu the challenge is that globals is not in `eng/common` so it can't be referenced by templates that are in `eng/common`. I think we would probably would want separate globals files, as the configurations we use for tools might be different than the ones we use for the language repos (and we may want to be able to iterate on those without having to do as much cross-pipeline testing).